src/support/azext_support/custom.py

```diff
@@ -272,3 +280,102 @@ def pre_operations(self):
         super().pre_operations()
         args = self.ctx.args
         _check_name_availability_no_subscription(self.cli_ctx, str(args.communication_name), "Microsoft.Support/communications")
+
+class FileWorkspaceCreateNoSubscription(_CreateNoSubscriptionFileWorkspace):
+    def pre_operations(self):
+        from azext_support._validators import _check_name_availability_no_subscription
+        super().pre_operations()
+        args = self.ctx.args
+        file_workspace_name = str(args.file_workspace_name)
+        _check_name_availability_no_subscription(self.cli_ctx, file_workspace_name, "Microsoft.Support/workspace")
+    
+class FileWorkspaceCreateSubscription(_CreateFileWorkspace):
+    def pre_operations(self):
+        from azext_support._validators import _check_name_availability_subscription
+        super().pre_operations()
+        args = self.ctx.args
+        file_workspace_name = str(args.file_workspace_name)
+        _check_name_availability_subscription(self.cli_ctx, file_workspace_name, "Microsoft.Support/workspace")
+    
+def encode_string_content(chunk_content):
```

Comment:
All these methods need to go into a separate helper function

Reply:
Not sure why this conversation was marked as resolved, we have a separate utils.py class, please move these helper functions to that class